css/style.css

```
@@ -1116,6 +1116,26 @@ h2 a {
   font-size: 16px;
 }
 
+/* h2 -> h3 header links */
+.heading-link::after {
+  content: '#';
+  color: var(--card-fg);
+}
+
+.heading-link {
+  margin-left: 0.3em;
+  font-size: 1rem;
+  text-decoration: none;

```

```suggestion
  text-decoration: none;
  color: inherit;
```

Nit: We can move styling of the pseudo element (the `::after` with `#`) color up a level. so we style the `a` itself to inherit from the header, and the pseudo element will inherit from there.

(will bork anything where the header is not the one providing a color, but that's useful in its own way)

---

yes